keanu-python/nd4j/build.gradle

```diff
@@ -0,0 +1,35 @@
+apply plugin: "java"
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile 'org.nd4j:nd4j-native-platform:1.0.0-beta3'
+}
+
+task copyNd4jJars(type: Copy) {
```

Comment:
It might be worth calling this `copyJarsIntoPythonClasspath` to match the name of the similar task in `keanu-project`